### DISCUSSION TITLE
Dataset which extends beyond Europe receives spatial reference system error

### DISCUSSION THREAD

**Discussion by Kate-Lyndegaard** [INSPIRE-MIF/helpdesk-validator#1250] [Ideas]
Hi,

I am receiving an error in rs.a.1: Spatial reference systems in feature geometries, however the Environmental Monitoring Facilities and Network dataset includes features outside of Europe. The dataset is provided in EPSG:4326 because in IR Requirement Annex II, Section 1.3.4 Coordinate Reference Systems it is stated that: 

"Exceptions, where other coordinate reference systems than those listed in 1.3.1, 1.3.2 or 1.3.3 may be used, are: 
1. Other coordinate reference systems may be specified for specific spatial data themes in this Annex. 
2. For regions outside of continental Europe, Member States may define suitable coordinate reference systems."

**WFS service:** https://haleconnect.com/ows/services/org.874.512633c0-2899-4f9d-b2d1-1bd51cb2315a_wfs?SERVICE=WFS&REQUEST=GetCapabilities&VERSION=2.0.0

**INSPIRE Validator Test Report:**
[WP4Dataset.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/6042729/WP4Dataset.zip)

We feel that this is a valid implementation, and I would appreciate your review of the issue. 

Kind regards,
Kate

**Comment by carlospzurita** [comment]
Dear @Kate-Lyndegaard 

Thank you for opening the issue. We are reviewing the list of supported EPSG and checking the IR for this. After our internal discussion, we will get back to you with some more feedback.

**Comment by carlospzurita** [comment]
Dear @Kate-Lyndegaard 

We are adding a check for datasets to control if they contain features from outside of continental Europe. If all features are located inside the boundary of Europe, then the only reference systems allowed are the ones defined on the IR. If it is not the case, then any reference system can be used.

We will get back to you whenever this fix is ready.

**Comment by Kate-Lyndegaard** [comment]
Hi @carlospzurita,

Thank you very much. 

Kind regards,
Kate

**Comment by fabiovinci** [comment]
Dear all,
since this issue requires the development of new features that are currently unplanned, it will be moved to the related discussion section.
